packages/hmr-plugin/src/utils/internals.ts

```diff
@@ -0,0 +1,15 @@
+import { HMR_RUNTIME } from '../symbols';
+
+declare const window: any;
+
+export function hmrSetReloaded(value: boolean = true): void {
```

Comment:
No need for `:boolean` as it's inferred automatically, `setHrmReloadedTo('qweqwe')` will throw.
```suggestion
export function setHmrReloadedTo(value = true): void {
```

Reply:
done